.github/workflows/test.yml

```diff
@@ -1,6 +1,14 @@
 name: "CI"
 
-on: push
+# The workflow should be triggered on any push and release events.
+# Release events can push tags (triggering a push event).
+# Therefore:
+#  - docker-publish-staging – is only triggered on push events to main branch
+#  - docker-publish-release – is only triggered on release events
+on:
+  push:
```

Comment:
Will you skip `pull_request` for now? This would leave open again to the situation where inbound changes from a fork won't trigger the workflow, unless I am missing something.

Reply:
Yes for now I'd not change this but this is something that we can definitely consider to add as a part of a separate issue 👍